config/samples/instana_v1_instanaagent.yaml

```diff
@@ -16,3 +16,8 @@ spec:
     configuration_yaml: |
       # You can leave this empty, or use this to configure your instana agent.
       # See https://docs.instana.io/setup_and_manage/host_agent/on/kubernetes/
+  # Example on how to limit Agent Resources
```

Comment:
Are we sure these are correct and will be supported in the future.
If I look at the helm chart, they are referenced like this:
https://github.ibm.com/instana/instana-agent-charts/tree/main/canonical#configuration-reference:~:text=agent.pod.limits,512Mi


```
agent.pod.limits.cpu
agent.pod.limits.memory
agent.pod.requests.cpu
agent.pod.requests.memory
```

Reply:
Interesting, I found this information on the operator and I applied the changes in my the monitoring without errors.
https://github.com/instana/instana-agent-operator/blob/main/config/crd/bases/instana.io_agents.yaml